website/blog/2018-12-14-iron.md

```diff
@@ -0,0 +1,89 @@
+---
+author: Ólafur Páll Geirsson
+title: Metals v0.3.2
+authorURL: https://twitter.com/olafurpg
+authorImageURL: https://avatars2.githubusercontent.com/u/1408093?s=460&v=4
+---
+
+We are pleased to announce the release of Metals v0.3.2. Metals is a language
+server for Scala that works with VS Code, Atom, Vim, Sublime Text and Emacs.
+Metals is developed at the [Scala Center](https://scala.epfl.ch/) with
+contributors from the community.
+
+<!-- truncate -->
+
+## Metals server
+
+Full details: https://github.com/scalameta/metals/milestone/3?closed=1
+
+In this release we closed 7 issues and merged 17 PRs.
+
+### Improved sbt launcher
+
+Metals now respects `.sbtopts` and `.jvmopts` when running `sbt bloopInstall`.
+If you need further customization like reading environment variables, then you
+can configure the "Sbt Script" setting (`-Dmetals.sbt-script=/path/to/sbt`).
+
+![sbt script](https://user-images.githubusercontent.com/1408093/50009698-e4f29000-ffb7-11e8-8d05-acb11f575748.png)
+
+### Build server discovery
+
+Metals now works with any build tool that implements
+["BSP Connection Protocol"](https://github.com/scalacenter/bsp/blob/master/docs/bsp.md#bsp-connection-protocol).
+See the Metals docs for more information:
+https://scalameta.org/metals/docs/build-tools/new-build-tool.html#custom-build-server
+
+### Blog post on fast goto definition with low memory footprint
+
+New blog post on how Metals implements Goto Definition:
+https://scalameta.org/metals/blog/2018/12/12/fast-goto-definition.html
+
+## VS Code
+
+Release notes: https://github.com/scalameta/metals-vscode/releases/tag/v1.1.0
+
+### Improved Java home detection
+
+The extension now automatically detects a Java 8 home directory even if Java 11
+is installed on the computer. Also, the extension now works for `jenv` users.
+
+### Automatic reload window detection
+
+The extension now automatically detects when "Reload window" is needed to pick
```

Comment:
I think this was in before.

The improvement in this release is about detecting outdated versions.

Reply:
Updated!